addons/website/models/website.py

```diff
@@ -635,6 +635,8 @@ def image_url(self, cr, uid, record, field, size=None, context=None):
         size = '' if size is None else '/%s' % size
         return '/website/image/%s/%s/%s%s' % (model, id, field, size)
 
+    def google_analytics_data(self, cr, uid, main_object, context=None):
```

Comment:
Is it necessary to define a void method here ? Website should be independent of any other module that override it; I think website_version should define this method, not override it. Is there any reason not doing it that way ?